packages/shared/invokeGuardedCallbackImpl.js

```diff
@@ -138,7 +133,15 @@ if (__DEV__) {
         ) {
           window.event = windowEvent;
```

Comment:
I wonder why this doesn't do

```js
if (windowEventDescriptor) {
    Object.defineProperty(window, 'event', windowEventDescriptor);
}
```

as below.

Reply:
I'm scared to touch it. https://github.com/facebook/react/pull/11696#discussion_r153686685